### ISSUE TITLE
Mallet toe deformity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11196]
**Preferred term label:**
Mallet toe deformity

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/?term=C0085686
Toe deformity characterized by distal interphalangeal (DIP) joint flexion with neutral alignment of the proximal interphalangeal (PIP) joint and metatarsophalangeal (MTP) joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mallet toe

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001297
- label: Mallet toe
- parents: [Toe deformity (HP:0034398)](https://hpo.jax.org/app/browse/term/HP:0034398)
- pmid: PMID:32211125
- def: A mallet toe involves the distal interphalangeal (DIP) joint [such that] the distal phalanx is flexed on the middle phalanx.
- comment: The etiology of mallet toe is varied, and has been attributed to anatomy and toe length, shoewear, trauma, or a factor in systemic neurologic disease. With a fixed mallet toe deformity, patients often complain of pain at the dorsal DIP joint, and/or pain at the tip of the affected toe, where undue pressure from footwear and weight bearing is experienced. Calluses may be found at either of these locations. When symptoms can no longer be tolerated with treatments such as modified shoewear and padding in a patient without underlying systemic disease, surgical intervention may be indicated.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11196
- synonyms:
